Week3/joyeeon/Mission1/src/components/moviescomp/MovieList.tsx

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import MovieItem from "./MovieItem";
+import { Movie } from "../../types/movie";
+import useLoading from "../../hooks/UseLoading";
+import IsLoading from "../state/IsLoading";
+import IsError from "../state/IsError";
+
+type Category = 'popular' | 'top_rated' | 'upcoming' | 'now_playing';
+
+interface CategoryProps {
+    category:Category;
+    page:number;
+    setTotalPages:(total:number) =>void;
+}
+
+const MovieList = ({category, page, setTotalPages} : CategoryProps) => {
+    const [movies, setMovies] = useState<Movie[]>([]);
+    const token = import.meta.env.VITE_TMDB_TOKEN;
+    
+
+    const fetchMovies = async () => {
+        const { data } = await axios.get(
+            `https://api.themoviedb.org/3/movie/${category}?language=ko-KR&page=${page}`,
+            {
+                headers: {
+                    Authorization: token,
```

Comment:
OAuth 2.0 표준에 따라 토큰 타입을 명시하는 "Bearer"를 접두어로 붙여주세요!

```suggestion
                    Authorization: `Bearer ${token}`,
```